src/settings.py

```diff
@@ -25,6 +25,9 @@ class GlobalSettings:
     video_dir = "videos"
     video_fps = 2
 
+    # teacher dataset parameters
+    frac = 0.3
```

Comment:
Can we make this name more specific? Maybe `teacher_dataset_label_ratio`? Or if that's too long, just `teacher_dataset_frac`?